File: 1895-minimum-number-of-operations-to-move-all-balls-to-each-box/1895-minimum-number-of-operations-to-move-all-balls-to-each-box.cpp
class Solution {
public:
    vector<int> minOperations(string boxes) {
        vector<int> v;
        
        int n = boxes.size();
        for(int i =0;i<n;i++){
            int sum =0;
            for(int j=0;j<n;j++){
                if(i==j){
                   continue;
                }
                if(boxes[j]=='1'){
                    sum+=abs(i-j);

                }

            }
            v.push_back(sum);
            
        }
        return v;
        
    }
};

File: 1895-minimum-number-of-operations-to-move-all-balls-to-each-box/1895-minimum-number-of-operations-to-move-all-balls-to-each-box.py
class Solution:
    def minOperations(self, boxes: str) -> List[int]:
        a=[]
        n=len(boxes)
        for i in range (n):
            sumi=0
            for j in range (n):
                if(i==j):
                    continue
                if(boxes[j]=='1'):
                    sumi+=abs(i-j)
            a.append(sumi)
        return a
        